mandatory/2-function-creation.js

```diff
@@ -16,7 +23,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (typeof num === "number") {
```

Comment:
Here, we already know that the argument to this function will be a number as it's in the description above, so we don't really need to check it again.